testsuite/tests/kuadrant/authorino/operator/tls/conftest.py

```diff
@@ -3,8 +3,27 @@
 
 import pytest
 
-from testsuite.certificates import CFSSLClient, Certificate
+from testsuite.certificates import CFSSLClient, Certificate, CertInfo
 from testsuite.openshift.httpbin import TLSEnvoy
+from testsuite.utils import cert_builder
+
+
+@pytest.fixture(scope="session")
+def certificates(cfssl, authorino_domain, wildcard_domain):
+    """All certificates that will be needed, formatted as a dict"""
```

Comment:
Please add more information to the doc. 
- depends on the order of the certificates